### PR TITLE
VE-2075: Massive cleanup to EBS - and actually get it to work

### PR DESCRIPTION
**Pull request by inez**

### CODE REVIEW

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 1]
```diff
@@ -1,67 +1,66 @@
...
 
-	protected function canOpenEBS( ) {
-		global $wgUser;
-		$ebs_response = $wgUser->getOption( 'ebs_response' );
-		if ($ebs_response === null) {
-			// user has not seen/interacted with EBS yet, so display it
-			return true;
-		} elseif ($ebs_response === 0) {
-			// user has clicked 'no', don't display it
+		$ebs_response = $user->getOption( self::EBS_RESPONSE_KEY );
```
Comment (pauloslund) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php]: These variables should be camelCased I believe.

**Comment by pauloslund** [comment]
Looks good to me

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 2]
```diff
@@ -1,67 +1,66 @@
...
-		$option_name = 'ebs_response';
-		if ($val === null) {
-			// invalid call, do nothing
-			wfErrorLog( 'Invalid call to EmergencyBroadcastSystemController::saveUserResponse()', $wgDebugLogFile );
-		} elseif ($val === 0) { // no
+	public function saveUserResponse( $val ) {
+		$user = $this->context->getUser();
+		$data = $this->request->getParams();
+
+		if ( $data[ 'val']  === '0' ) { // no
```
Comment (katanka) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php:22]: spaces

File: extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php [hunk 3]
```diff
@@ -1,67 +1,66 @@
...
 			// if user clicks no, set ebs_response to 0
-			$wgUser->setOption( $option_name, 0 );
-		} else {
+			$user->setOption( self::EBS_RESPONSE_KEY, 0 );
+		} elseif ( $data[ 'val' ] === '1' ) { // yes
 			// if user clicks yes, set ebs_response to the current time
-			$timestamp = (new DateTime())->getTimestamp();
-			$wgUser->setOption( $option_name, $timestamp );
+			$user->setOption( self::EBS_RESPONSE_KEY, ( new DateTime() )->getTimestamp() );
+		} else if ( $data[ 'val']  === '-1' ) { // for testing purposes
```
Comment (katanka) [inline extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php:28]: spaces

**Comment by katanka** [comment]
Spacing is weird in a few places, but otherwise LGTM

**Pull request merged**